### DISCUSSION TITLE
음성 채팅 기능 지원 여부

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#973] [5. Q&A]
안녕하세요 :)

월드에 음성 채팅 기능을 사용하려 합니다.
혹시 뚫려있는 메서드가 있을까요 ?

없다면 혹시 차후 제공 해주실 예정이신지 궁금합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 음성 채팅 기능은 준비 중에 있으며 하반기 중에는 지원하는 것을 목표로 하고 있습니다.
조만간 로드맵이 공개될 예정이오니 새로운 기능들을 확인해 보시고 의견도 많이 주세요.
감사합니다.
